<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSpeedDial</class>
 <widget class="QDialog" name="DlgSpeedDial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Speed Dial</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>170</width>
     <height>346</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblF1">
        <property name="text">
         <string> 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblF2">
        <property name="text">
         <string> 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblF3">
        <property name="text">
         <string> 3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblF4">
        <property name="text">
         <string> 4</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblF5">
        <property name="text">
         <string> 5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblF6">
        <property name="text">
         <string> 6</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblF7">
        <property name="text">
         <string> 7</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblF8">
        <property name="text">
         <string> 8</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lblF9">
        <property name="text">
         <string> 9</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboF1"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboF2"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboF3"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboF4"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboF5"/>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboF6"/>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboF7"/>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboF8"/>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboF9"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbOk">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgSpeedDial</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
